File: misc/versus/javascript/zhongwen.js
//- JavaScript source code

//- zhongwen.js ~~
//                                                      ~~ (c) SRW, 09 Aug 2011

chassis(function (q) {
    "use strict";

    q.lib("base") || q.die();

    var decoded, message;

    message = [20320, 22909, 21527, 63];
    decoded = q.base$map(message).using(String.fromCharCode);

    q.puts(decoded.join(""));

});

//- vim:set syntax=javascript:

File: misc/versus/go/zhongwen.go
//- Go source code

//- zhongwen.go ~~
//                                                      ~~ (c) SRW, 09 Aug 2011

package main

import (
    "flag"
    "fmt"
)

func main() {

    flag.Parse()

    message := [...]int {20320, 22909, 21527, 63}
    decoded := ""

    for _, val := range message {
        decoded += fmt.Sprintf("%c", val)
    }

    print(decoded, "\n")

}

//- vim:set syntax=go:
